Makefile

```diff
@@ -31,6 +31,8 @@ spellcheck:
 
 check:
 	pdm run pre-commit run --all-files
+	pdm export -o requirements/docker.txt --without-hashes --prod
+	pdm export -o requirements/test.txt --without-hashes --dev
```

Comment:
These would not be needed if we can get a pdm `post_lock` hook to run the exports automatically.